client/components/PhotogridComponents/Photogrid.jsx

```diff
@@ -20,6 +20,11 @@ const Big = styled.div`
   grid-area: 1 / 1 / 3 / 3;
   border-top-left-radius: 15px;
   border-bottom-left-radius: 15px;
+  &:hover {
+    -webkit-filter: grayscale(50%);
+    filter: brightness(80%);
+    opacity: 0.8;
+  }
 `;
 
 const Photo1 = styled.div`
```

Comment:
when you have time, could you think of a way to refactor and write less code? It seems as though it's redundant. However, great on organizing your styling into different component and making it easy to read and follow

Reply:
Yeah i def want to! Isn't very dry rn. Just gotta figure out a way to have each grid elemenet be positioned correctly using props.